draft-ietf-taps-impl.md

```diff
@@ -595,7 +595,10 @@ It is also possible for Protocol Stacks within a particular leaf node to use 0-R
 
 Message Framers are functions that define
 simple transformations between application Message data and raw transport
-protocol data. A Framer can encapsulate or encode outbound Messages,
+protocol data. Generally, a Message Framer implements a simple
+application protocol that can either be provided by the Transport Service
+implementation or be customized by the application.
```

Comment:
```suggestion
implementation or by the application. It is optional for Transport Services system implementations to provide Message Framers: the specification {{I-D.ietf-taps-interface}} does not prescribe any particular Message Framers to be implemented.
```
1) this suggests "by the application" because "customized" sounds like the application could only adjust an offered Framer, not entirely implement one (as it would).
2) the rest is a proposal to make it clear that no framers must be shipped with a TAPS system... to better address #1145.